Library/Formula/libjit.rb

```diff
@@ -0,0 +1,19 @@
+class Libjit < Formula
+  homepage "https://www.gnu.org/software/libjit/"
+  url "git://git.savannah.gnu.org/libjit.git"
+  version "0.1.3"
+
+  depends_on "gcc" => :build
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+
+  def install
+    gcc = Formula["gcc"]
+    system "./auto_gen.sh"
+    system "./configure", "--prefix=#{prefix}",
+                          "CC=#{gcc.bin}/gcc-#{gcc.version_suffix}"
+    system "make"
+    system "make install"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!